src/main/java/com/checkmarx/sdk/service/CxService.java

```diff
@@ -755,6 +755,8 @@ private boolean filterOsa(List<Filter> filters, CxOsa osa) {
                         if(!resultType.getFalsePositive().equalsIgnoreCase("FALSE")){
                             falsePositive = true;
                         }
+                        log.debug("getIssues: id: {}, severity: {}, result type filename: {}, result type severity: {}",
```

Comment:
This message will add quite a lot of logging. Are you sure you want to have it here?
I remember there is a similar per-issue log somewhere in CxFlow ResultsService during publishing.

Reply:
This was left over from when I was testing the change. I will remove it.